dataset.py

```diff
@@ -560,9 +561,11 @@ def load_and_scale(config: Dict,
                    extra_columns_from_geotiffs: Dict[str, raster.AmazonGeoTiff]) -> ScaledPartitions:
   columns_to_keep = columns_to_passthrough + columns_to_scale + columns_to_standardize
   X_train, Y_train = load_dataset(config['TRAIN'], mean_label, var_label, columns_to_keep, extra_columns_from_geotiffs)
-  X_val, Y_val = load_dataset(config['VALIDATION'], mean_label, var_label, columns_to_keep, extra_columns_from_geotiffs)
   X_test, Y_test = load_dataset(config['TEST'], mean_label, var_label, columns_to_keep, extra_columns_from_geotiffs)
-
+  
+  # Optionally load validation dataset. 
+  X_val, Y_val = None, None if 'VALIDATION' not in config else load_dataset(config['VALIDATION'], mean_label, var_label, columns_to_keep, extra_columns_from_geotiffs)
+  
```

Comment:
When using cross-validation, would it also make sense for the test set to be optional?